src/dlgTriggerEditor.cpp

```diff
@@ -1111,21 +1111,10 @@ void dlgTriggerEditor::slot_search_triggers( const QString s )
             {
                 QStringList sl;
                 sl << "Key" << key->getName() << "name";
-                if( ! parent )
```

Comment:
This was kind of confusing because I thought that expansion was getting lost with the removal of the `setExpanded()` call, but it's later down and in tests the behaviour is still the same.